contrib/trace_propagators/src/main/java/io/opentelemetry/contrib/trace/propagation/JaegerPropagator.java

```diff
@@ -39,7 +39,7 @@
 
 /**
  * Implementation of the Jaeger propagation protocol. See <a
- * href=https://www.jaegertracing.io/docs/client-libraries/#propagation-format>Jaeger Propogation
+ * href=https://www.jaegertracing.io/docs/client-libraries/#Propogation-format>Jaeger Propogation
```

Comment:
Is the link broken?

Reply:
My bad, I wanted to change Propogation to Propagation. Corrected now, please review again